docs/manifest.json

```diff
@@ -143,60 +143,70 @@
-                        {
-                          "title": "Updating Projects",
-                          "path": "/docs/portal/image-registry/updating-projects.md"
-                        },
-                        {
-                          "title": "Deleting Projects",
-                          "path": "/docs/portal/image-registry/deleting-projects.md"
-                        },
-                    {
-                      "title": "Managing Registry Users",
-                      "path": "/docs/portal/registry-users/managing-registry-users.md"
-                    },
-                         {
-                           "title": "Adding New Users",
-                           "path": "/docs/portal/registry-users/adding-users.md"
-                         },
-                         {
-                          "title": "Editing Users",
-                          "path": "/docs/portal/registry-users/editing-users.md"
-                        },
-                        {
-                          "title": "Changing Passwords",
-                          "path": "/docs/portal/registry-users/changing-passwords.md"
-                        },
-                        {
-                          "title": "Updating Users",
-                          "path": "/docs/portal/registry-users/updating-users.md"
-                        },
-                        {
-                          "title": "Deleting Users",
-                          "path": "/docs/portal/registry-users/deleting-users.md"
-                        }
-                      ]
-                    },
+                "title": "Managing Registry Projects",
+                "path": "/docs/portal/image-registry/managing-registry-projects.md"
+                },
+                {
+                 "title": "Adding New Projects",
+                 "path": "/docs/portal/image-registry/adding-projects.md"
+                },
+                {
+                 "title": "Viewing Projects",
+                 "path": "/docs/portal/image-registry/viewing-projects.md"
+                },
+                {
+                 "title": "Viewing Logs",
+                 "path": "/docs/portal/image-registry/viewing-logs.md"
+                },
+                {
+                 "title": "Managing Repositories Under a Project",
+                 "path": "/docs/portal/image-registry/managing-repositories-under-a-project.md"
+                },
+                {
+                  "title": "Managing Tags under a Repository",
+                  "path": "/docs/portal/image-registry/managing_tags-under-a-repository.md"
+                },
+                {
+                  "title": "Updating Projects",
+                  "path": "/docs/portal/image-registry/updating-projects.md"
+                },
+                {
+                 "title": "Deleting Projects",
+                 "path": "/docs/portal/image-registry/deleting-projects.md"
+                }
```

Comment:
A comma to delimit 2 objects is missing here.

Reply:
Thank you. I fixed this and bookmarked the URL for future reference.